vispy/visuals/shaders/function.py

```diff
@@ -430,7 +431,7 @@ def __setitem__(self, key, val):
                     self.template_vars.add(var.lstrip('$'))
         
         self.changed(code_changed=True, value_changed=True)
-        if logger.level >= 10:
+        if logger.level == logging.DEBUG:
```

Comment:
I think this could be `<= logging.DEBUG` in case we ever make/use a custom logging level that is below DEBUG